sql/moz-fx-data-shared-prod/mozilla_vpn_derived/site_metrics_summary_v1/query.sql

```diff
@@ -88,7 +85,12 @@ base_table AS (
       STRUCT(campaign, content, medium, source)
     ).*,
   FROM
-    ga_sessions_hits
+    ga_sessions
+  CROSS JOIN
+    UNNEST(hits)
+  WHERE
+    `date` IS NOT NULL
+    AND `date` = @date
```

Comment:
I'm also curious if this will properly push down partition filters in BigQuery, but it's not clear to me that this would have been the case in the previous version either.

Reply:
dry run estimates indicate that it works properly (both before and after this change)